libs/database/src/lib/entities/delayed-job.entity.ts

```diff
@@ -1,7 +1,24 @@
-import { AllowNull, AutoIncrement, Column, Default, ForeignKey, Index, Model, PrimaryKey, Table } from "sequelize-typescript";
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Default,
+  ForeignKey,
+  Index,
+  Model,
+  PrimaryKey,
+  Table
+} from "sequelize-typescript";
 import { BIGINT, BOOLEAN, INTEGER, JSON, STRING, UUID } from "sequelize";
 import { User } from "./user.entity";
+import { Site } from "./site.entity";
+import { Project } from "./project.entity";
+import { Logger } from "@nestjs/common";
 
+const ENTITY_TYPE_MAPPING: Record<string, any> = {
```

Comment:
We try very hard to avoid `any` in this codebase. I know this kind of map is hard to create with correct types... fortunately, there's a better solution outlined below.

Reply:
Removed the use of entity type mapping